mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -619,6 +651,31 @@ class DistributedLDAModel private[ml] (
   @Since("1.6.0")
   lazy val logPrior: Double = oldDistributedModel.logPrior
 
+  private var _checkpointFiles: Array[String] = oldDistributedModel.checkpointFiles
+
+  /**
+   * If using checkpointing and [[LDA.keepLastCheckpoint]] is set to true, then there may be
+   * saved checkpoint files.  This method is provided so that users can manage those files.
+   * Note that removing the checkpoints can cause failures if a partition is lost and is needed
+   * by certain [[DistributedLDAModel]] methods.
+   *
+   * @return  Checkpoint files from training
+   */
+  @Since("2.0.0")
+  def getCheckpointFiles: Array[String] = _checkpointFiles
```

Comment:
Do we really want to expose this? It seems somewhat closely tied to the implementation - maybe we should mark this as a developer API?